### PR TITLE
Accueil page

### PR DESCRIPTION
**Pull request by ozen2**

### CODE REVIEW

File: client/src/components/Footer.jsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import ButtonContact from "./ButtonContact";
+import Links from "./Links";
 import "../styles/Footer.scss";
 
 function Footer() {
   return (
-    <footer>
-      <ButtonContact />
+    <>
```
Comment (Yoann-Fortin) [inline client/src/components/Footer.jsx]: tag footer is better here

File: client/src/components/HomeCards.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  imageMid,
+  imageRight,
+}) {
+  return (
+    <>
+      <h2 className="sectionTitle">{sectionTitle}</h2>
+      <article className="allCards">
+        <section className="Cards">
+          <figcaption>
+            <img id="left" src={imageLeft} alt="" />
```
Comment (Yoann-Fortin) [inline client/src/components/HomeCards.jsx]: Why id attribut ,

File: client/src/components/HomeCards.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            <img id="mid" src={imageMid} alt="" />
+            <img id="right" src={imageRight} alt="" />
+          </figcaption>
+          <hgroup>
```
Comment (Yoann-Fortin) [inline client/src/components/HomeCards.jsx]: Wrong uses tag here

File: client/src/components/Slogan.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import "../styles/Slogan.scss";
+
+function Slogan() {
+  return (
+    <hgroup className="slogan">
```
Comment (Yoann-Fortin) [inline client/src/components/Slogan.jsx]: wrong uses tag

File: client/src/components/TopPlanet.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import PropTypes from "prop-types";
+import Links from "./Links";
+import "../styles/Header.scss";
+
+function TopPlanet({ introPlanet, planetName }) {
+  return (
+    <section className="titleSlider">
+      <h2>{introPlanet}</h2>
```
Comment (Yoann-Fortin) [inline client/src/components/TopPlanet.jsx:8]: don't use tag for style

File: client/src/styles/HomeCards.scss [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    height: 14rem;
+    align-items: center;
+
+    figcaption {
+      display: flex;
+      flex-direction: row;
+
+      img {
+        width: 5.65rem;
+        border-left: solid 0.1rem white;
```
Comment (Yoann-Fortin) [inline client/src/styles/HomeCards.scss]: Don't you have color variables?

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: client/src/components/NavBar.jsx [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+function NavBar() {
+  const [burgerClass, setBurgerClass] = useState("burger-bar unclicked");
+  const [menuClass, setMenuClass] = useState("menu hidden");
+  const [isMenuClicked, setIsMenuClicked] = useState(false);
+
+  const menuRef = useRef(null);
+
+  useEffect(() => {
+    const handleTabNavigation = (e) => {
```
Comment (Yoann-Fortin) [inline client/src/components/NavBar.jsx:13]: Can you explain me this useEffect ?

File: client/src/styles/Header.scss [hunk 8]
```diff
@@ -0,0 +1,51 @@
+@use "./app.scss" as *;
```
Comment (Yoann-Fortin) [inline client/src/styles/Header.scss:1]: Can you explain me this line ?

File: client/src/styles/Slogan.scss [hunk 9]
```diff
@@ -0,0 +1,36 @@
+.slogan {
+  padding-top: 5rem;
+  padding-bottom: 6rem;
+  margin-left: 1rem;
+
+  h3 {
+    font-size: 2rem;
+    color: #9548f9;
```
Comment (Yoann-Fortin) [inline client/src/styles/Slogan.scss]: Don't have globas css of variable ?

File: client/src/styles/app.scss [hunk 10]
```diff
@@ -1,5 +1,80 @@
+@import url("https://fonts.googleapis.com/css2?family=Noto+Sans+Sundanese:wght@400..700&family=Tomorrow:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Varela+Round&display=swap");
+
+$whiteColor: #fff8f0;
```
Comment (Yoann-Fortin) [inline client/src/styles/app.scss:2]: Why don't use this variables css ?

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Review by pjroggy — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**